dds/DCPS/SporadicTask.h

```diff
@@ -25,7 +25,7 @@ class SporadicTask : public RcEventHandler {
     : time_source_(time_source)
     , interceptor_(interceptor)
     , desired_scheduled_(false)
-    , actual_scheduled_(false)
+    , timer_id_(0)
```

Comment:
Do timer ids start at 1?

Reply:
Nope... I was thinking of the timers in EventDispatcher.